php/libraries/Utility.class.inc

```diff
@@ -150,6 +151,39 @@ class Utility
         return $list;
     }
 
+    /**
+     * Returns a list of sites by site IDs from the database
+     *
+     * @param array $siteIDs a list of the study center IDs
+     *
+     * @return array an associative array("center ID" => "site name")
+     */
+    static function getSiteListByIDs(array $siteIDs): array
```

Comment:
@driusan shouldnt a function like this exist in the site class rather than utility ?